src/Umbraco.Infrastructure/Runtime/RuntimeState.cs

```diff
@@ -225,7 +225,12 @@ public void DetermineRuntimeLevel()
 
                     // else it is bad enough that we want to throw
                     Reason = RuntimeLevelReason.BootFailedCannotConnectToDatabase;
-                    BootFailedException = new BootFailedException("A connection string is configured but Umbraco could not connect to the database.");
+                    BootFailedException = new BootFailedException(
+                        $"""
+                         A connection string is configured but Umbraco could not connect to the database.
+                         Possible causes include network errors, a bad connection string or a missing database.
+                         Tip: Use {nameof(GlobalSettings)}:{nameof(GlobalSettings.InstallMissingDatabase)} to instruct Umbraco to create the database automatically.
```

Comment:
This should contain the complete setting name (including `Umbraco:CMS:`), which is available as a constant:
```suggestion
                         Tip: Use {Constants.Configuration.ConfigGlobal}:{nameof(GlobalSettings.InstallMissingDatabase)} to instruct Umbraco to create the database automatically.
```